keyboards/wete/rules.mk

```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = STM32F072
```

Comment:
Which F072 chip is this, specifically?

#8179

Reply:
This is the STM32F072CBT6. Should I replace this with `STM32F07xB`?